src/generators/results.ts

```
@@ -552,19 +579,61 @@ ${allFields.join(',\n')}
   }
 
   private mapPrismaTypeToZod(field: DMMF.Field): string {
+    const isJsonSchemaCompatible = this.isJsonSchemaModeEnabled();
+
+    // Handle JSON Schema compatibility mapping
+    if (isJsonSchemaCompatible) {
+      switch (field.type) {
+        case 'DateTime':
+          const { dateTimeFormat } = this.getJsonSchemaOptions();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)